### PR TITLE
[MAINT] add citation file and validation workflow

### PR DESCRIPTION
**Pull request by Remi-Gau**
- add a citation.cff file to the repo with metadata about contributors
- add a workflow to make sure the file is valid

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    email:  nikhil.bhagwat@mcgill.ca
+    family-names: Bhagwat
+    given-names: Nikhil
+
+authors:
+  - affiliation: "ORIGAMI lab, McGill University, Montréal, Canada"
+    email:  nikhil.bhagwat@mcgill.ca
+    family-names: Bhagwat
+    given-names: Nikhil
+    orcid: "https://orcid.org/0000-0002-5276-8818"
```
Comment (nikhil153) [inline CITATION.cff:25]: This belongs to some `Alice Van Audenhaege`

Comment (nikhil153) [reply CITATION.cff:25]: This is me:
`0000-0001-6073-7141`

Comment (Remi-Gau) [reply CITATION.cff]: should be fixed now

**Comment by Remi-Gau** [comment]
I do not have merge rights on this so @nikhil153 I will let you handle this

**Review by nikhil153 — APPROVED** [review]

**Comment by nikhil153** [comment]
@Remi-Gau - Done!

**Comment by Remi-Gau** [comment]
Thanks

**Pull request merged**